spec.bs

```diff
@@ -376,7 +376,8 @@ The Shared Storage API will integrate into the [=Storage Model|Storage API=] as
 
     1. Let |origin| be |environment|'s [=url/origin=].
     1. If |origin| is an [=opaque origin=], then return failure.
-    1. If the user has disabled [=shared storage=], then return failure.
+    1. If the user has disabled [=shared storage=] or operation is denied due to other user agent defined mechanism e.g. |origin|'s site not being
```

Comment:
More precisely, I think this should be `|origin|'s [=origin/host=]'s [=host/registrable domain=]`?

Reply:
Since you add the link to multiple sites, it seems better to add the link to `<pre class="anchors">` section at the top, and reference it via `[=enrolled=]` in the main text.

```
html; urlPrefix: https://github.com/privacysandbox/attestation
    type: dfn
        text: enrolled
```

Also, I just noticed it's the schemeful site. So, this can be `e.g. the result of [=obtaining a site=] with |origin| not being [=enrolled=]`.

Reply:
(FYI I also made a PR to add the settings check for addModule/selectURL/run: https://github.com/WICG/shared-storage/pull/106)

Reply:
Edited to add this link.

Reply:
And used something similar to `the result of [=obtaining a site=] with |origin| not being [=enrolled=]`...

Could probably still be further edited. xyaoinum@ what do you think?